test/pylib/cpp/boost/boost_facade.py

```diff
@@ -99,13 +111,13 @@ def read_file(name: Path) -> str:
                 return ''
 
         timeout = TIMEOUT_DEBUG if mode=='debug' else TIMEOUT
-        root_log_dir = self.temp_dir / mode / 'pytest'
```

Comment:
Hm... And logs are now moved up one level? How does it help to gather resource metrics for C++ tests? This particular patch is huge enough to have much more descriptive comment that it has now, please write one.